src/components/response/ResponseBlock.tsx

```diff
@@ -150,14 +103,9 @@ export default function ResponseBlock({
         )}
         {showNextBtn && (
           <NextButton
-            disabled={
-              hasCorrectAnswer
-                ? !checkClicked
-                : !areResponsesValid
-            }
-            process={() => {
+            disabled={hasCorrectAnswer && !checkClicked}
```

Comment:
Maybe we could us another name instead of `disabled`. Perhaps `correctAnswerChecked` would be a better name